core/pom.xml

```diff
@@ -215,6 +215,11 @@
       <version>3.2.10</version>
     </dependency>
     <dependency>
+      <groupId>com.fasterxml.jackson.module</groupId>
+      <artifactId>jackson-module-scala_2.10</artifactId>
+      <version>2.3.1</version>
```

Comment:
this seems like a common dependency by our users. how well does jackson work against different versions?